crates/zkforge/bin/cmd/script/build.rs

```diff
@@ -211,63 +230,52 @@ impl ScriptArgs {
         })
     }
 
-    pub fn get_project_and_output(
+    pub async fn get_project_and_output(
         &mut self,
         script_config: &ScriptConfig,
     ) -> Result<(Project, ProjectCompileOutput)> {
         let project = script_config.config.project()?;
-
-        let filters = self.opts.skip.clone().unwrap_or_default();
-        // We received a valid file path.
-        // If this file does not exist, `dunce::canonicalize` will
-        // result in an error and it will be handled below.
-        if let Ok(target_contract) = dunce::canonicalize(&self.path) {
-            let output = compile::compile_target_with_filter(
-                &target_contract,
-                &project,
-                self.opts.args.silent,
-                self.verify,
-                filters,
-            )?;
-            return Ok((project, output))
-        }
-
-        if !project.paths.has_input_files() {
-            eyre::bail!("The project doesn't have any input files. Make sure the `script` directory is configured properly in foundry.toml. Otherwise, provide the path to the file.")
-        }
+        let mut zksolc_cfg = script_config.config.zk_solc_config().map_err(|e| eyre::eyre!(e))?;
 
         let contract = ContractInfo::from_str(&self.path)?;
         self.target_contract = Some(contract.name.clone());
 
-        // We received `contract_path:contract_name`
-        if let Some(path) = contract.path {
-            let path =
-                dunce::canonicalize(path).wrap_err("Could not canonicalize the target path")?;
-            let output = compile::compile_target_with_filter(
-                &path,
-                &project,
-                self.opts.args.silent,
-                self.verify,
-                filters,
-            )?;
-            self.path = path.to_string_lossy().to_string();
-            return Ok((project, output))
+        // A contract was specified by path
+        // TODO: uncomment the `if let` block once we support script by contract name
+        // if let Ok(_) = dunce::canonicalize(&self.path) {
+        let compiler_path = setup_zksolc_manager(DEFAULT_ZKSOLC_VERSION.to_owned()).await?;
+        zksolc_cfg.compiler_path = compiler_path;
+        zksolc_cfg.settings.is_system = false;
+        zksolc_cfg.settings.force_evmla = false;
```

Comment:
I dont think this needed as these values are default `false`

Reply:
Cleaned